app/controllers/api/v1/wallet_transactions_controller.rb

```diff
@@ -4,8 +4,7 @@ module Api
   module V1
     class WalletTransactionsController < Api::BaseController
       def create
-        service = WalletTransactions::CreateService.new
-        result = service.create(
+        result = WalletTransactions::CreateService.new.call(
```

Comment:
Can we shorten it to just:

```suggestion
        result = WalletTransactions::CreateService.call(
```

Reply:
Yes, that's the plan, but it requires several changes due to the fact that we have to refactor `WalletTransactions::ValidateService`.

So if that's ok for you, will do it in a future PR ;)